build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -xe
+
+SCALAR_CFSSL_IMAGE_TAG=1.0.0
+CFSSL_GIT_TAG=v1.4.1
```

Comment:
(IMO) 

- CFSSL_GIT_TAG=1.4.1

```
git clone --depth 1 --branch v${CFSSL_GIT_TAG} https://github.com/cloudflare/cfssl.git cfssl
WORKING_CFSSL_IMAGE=cloudflare-cfssl:${CFSSL_GIT_TAG}-alpine
```

Reply:
Thank you for the comment. I thought about the same thing. It's a kind of annoying problem. If we do like `CFSSL_GIT_TAG=1.4.1`, it is not a "git tag" any more, so we should name it like `CFSSL_IMAGE_TAG`. But, then the problem is that there is no guarantee that `v${CFSSL_IMAGE_TAG}` exists in the GitHub repo. I ended up the decision that we should specify the "existing" one, the git tag, then derive the Docker image tag from it.